### DISCUSSION TITLE
Change the TTS

### DISCUSSION THREAD

**Discussion by Abbadon999** [HRNPH/AIwaifu#21] [Ideas]
How do I change the main language? I also wouldn't mind changing the main TTS to TTS by [snakers4/silero-models](https://github.com/snakers4/silero-models), it's free and very easy to install, as well as changing the language. Silero also has English, so I would be much more comfortable using it than your TTS. Is it possible to change the TTS yourself or do you have plans to do it yourself?

**Comment by HRNPH** [comment]
# First of all, it is possible to change the TTS
well you could replace the whole folder of anime TTS with your code and named it custom_inference.py accept text and save voice with the same file name in the cache folder but for now there's no easy way to change it
# Secondly, I'm planning to change the whole TTS? YES!
the current TTS was intended to be used to prototyping and prove that this project is interesting enough(which it did)
The idea of our new TTS corpus is every advanced user can add their own TTS easily by following the same template
You can take a look at the file structure and the sample code I'm developing

![image](https://user-images.githubusercontent.com/51855316/231874416-37acc476-83e0-41d1-9526-ad4247b7dd15.png)

as for a more casual way of changing how our waifu sounds, we'll use [soft voice conversion](https://huggingface.co/spaces/skytnt/moe-tts)
which will be easier to train by yourself than TTS to transform the TTS voice in to other voice(I already test that it work well)
